settings.gradle

```diff
@@ -12,26 +13,30 @@ if (!JavaVersion.current().java11) {
 }
 
 gradle.ext.ciBuild = System.getenv()['JENKINS_URL'] ? true : false
+gradle.ext.ciTeamCityBuild = System.getenv('TEAMCITY_VERSION') ? true : false
 
 gradleEnterprise {
   buildScan {
-    if (gradle.ciBuild) {
-      tag('CI')
-      ifNonNull(System.getenv('GIT_BRANCH')) { tag(it) }
-      ifNonNull(System.getenv('GIT_COMMIT')) { value('git-commit-id', it) }
-      ifNonNull(System.getenv('BUILD_URL')) { link('Jenkins build', it) }
-
-      termsOfServiceUrl = 'https://gradle.com/terms-of-service'
-      termsOfServiceAgree = 'yes'
-      publishAlways()
-    } else {
-      tag 'LOCAL'
+    capture { taskInputFiles = true }
+    uploadInBackground = !gradle.ciBuild
```

Comment:
shouldn't this take `ciTeamCityBuild` into account too ?

Reply:
Done